### PR TITLE
Fixes an order issue with insert_at

### LINKED ISSUE #150: insert_at doesn't seem to be working in ActiveRecord callback (Rails 4.2)
Trying to reorder a list inside of an after_create callback doesn't seem to do any reordering.  I created a little example with an rspec test in it.

https://github.com/jasonsof/acts-as-list-example

Is this normal, or am I doing something wrong here?

### PR DESCRIPTION
**Pull request by joeadcock**
This fixes #150.  This was the behavior in version 0.4.0, but has been broken in subsequent versions.

### CODE REVIEW

File: lib/acts_as_list/active_record/acts/list.rb [hunk 1]
```diff
@@ -280,6 +280,7 @@ def default_position?
         # Sets the new position and saves it
         def set_list_position(new_position)
           write_attribute position_column, new_position
+          @position_changed = true if new_record?
           save(validate: false)
```
Comment (swanandp) [inline lib/acts_as_list/active_record/acts/list.rb]: `position_column` may or may not be `position`.

Comment (swanandp) [inline lib/acts_as_list/active_record/acts/list.rb]: Although I am wondering, why does this fix the bug?

**Comment by brendon** [comment]
Is this one still necessary? :)

**Comment by brendon** [comment]
Closing this due to inactivity. @joeadcock, let me know if you want it reopened so you can pursue this.

**Pull request closed without merging**